src/main/java/com/github/onsdigital/perkin/SurveyListener.java

```diff
@@ -141,7 +149,11 @@ public String test() throws IOException {
 
     @Override
     public void handleRecovery(Recoverable recoverable) {
-        log.warn("QUEUE|CONNECTION|END|queue connection was recovered");
+        log.warn("QUEUE|CONNECTION|END|queue connection was recovered: {}", recoverable);
```

Comment:
What should this log out? I get "...queue connection was recovered: com.rabbitmq.client.impl.recovery.AutorecoveringChannel@3fe0a7..."